bundles/org.openhab.core.io.rest.swagger1/.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>org.openhab.core.io.rest.optimize</name>
+	<name>org.openhab.core.io.rest.swagger1</name>
```

Comment:
Will we also have ever a swagger2 in parallel? If not, I'd prefer to simply call the bundle "swagger" here.

Reply:
It's not likely so I'll update it.